composer.json

```diff
@@ -5,8 +5,13 @@
 	"keywords": ["silverstripe", "cwp"],
 	"license": "BSD-3-Clause",
 	"require": {
-		"cwp/cwp-core": "*",
-		"silverstripe/taxonomy": "*"
+		"cwp/cwp-core": "^1",
+		"silverstripe/taxonomy": "^1",
+		"symbiote/silverstripe-advancedworkflow": "^4",
```

Comment:
We should, instead, make `.travis.yml` install the basic recipe than hard-linking these dependencies.